.env.example

```
@@ -1,5 +1,4 @@
-STRIPE_PUBLISHABLE_KEY = "pk_test_51JJ3vNLOlCyqSHKr5xPVUVHcaD9WBEkIsxic1lK4e9LNkfyYnZ3iYLwINAXw78bZfl3kqCY47EraOGPYbqtDxFeI00jgjkf0Kp"
+
 STRIPE_SECRET_KEY = "sk_test_51JJ3vNLOlCyqSHKrp2vu93c9YzrXxDq7otphDYZqQTE10eZ0GsiVPJPxEOYOHEIBsMW9qjvtIZGPM2pcJTaYulvE00MlBr7pi9"
```

```suggestion
TEST_STRIPE_SECRET_KEY = "sk_test_51JJ3vNLOlCyqSHKrp2vu93c9YzrXxDq7otphDYZqQTE10eZ0GsiVPJPxEOYOHEIBsMW9qjvtIZGPM2pcJTaYulvE00MlBr7pi9"
```

What do you think of prefixing the test key with `TEST_` so it's clear it's not the real live key?

---

Great idea! Implementing!